.idea/misc.xml

```diff
@@ -3,5 +3,4 @@
   <component name="JavaScriptSettings">
     <option name="languageLevel" value="ES6" />
   </component>
-  <component name="ProjectRootManager" version="2" project-jdk-name="Remote Python 3.7.3 Docker Compose (django at [/home/piotr/work/OpenCraft/projects/sprints/local.yml])" project-jdk-type="Python SDK" />
```

Comment:
@Agrendalath, are these (this and a few other changes below) related to the Google social auth setup at all? If so, can you separate them into a different commit that makes it obvious what they are for?

Reply:
@lgp171188, this has been pushed accidentally with #1 after renaming the whole project. Should I remove it from here and make a separate PR for this change?